src/tests/create_deck.test.js

```diff
@@ -0,0 +1,10 @@
+import { DECK } from "../utils/create_deck";
+
+describe("createDeck", () => {
+  it("creates a new deck", () => {
+    expect(DECK).toMatchSnapshot();
+  });
+  it("Checks if created a full deck", () => {
+    expect(DECK.length).toBe(52);
```

Comment:
Dica: você pode usar [. toHaveLength(number)](https://jestjs.io/pt-BR/docs/expect#tohavelengthnumber) aqui

```suggestion
    expect(DECK).toHaveLength(52);
```